ui/k-info-panel/show_circuit.kytos

```diff
@@ -782,53 +785,53 @@ module.exports = {
       }
 
       ['primary_constraints', 'secondary_constraints'].forEach(_type => {
-        payload[_type] = {};
+        let _constraint_payload = {}; 
         // Dropdown forms
-        payload[_type].desired_links = [];
-        if(this.constraints[_type].desired_links && this.constraints[_type].desired_links.length > 0) {
-          payload[_type].desired_links = [].concat(
+        if(this.constraints[_type].desired_links !== undefined && this.constraints[_type].desired_links.length > 0) {
```

Comment:
`desired_links` is no longer supported, could you merge `master` into your branch and solve the git conflicts?